docs/DeveloperGuide.adoc

```diff
@@ -294,72 +296,107 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a(n) ... |I want to ... |So that I can...
+|`* * *` |social student |keep track of who owes me money & how much | not have anyone owe me any money.
 
-|`* * *` |user |add a new person |
+|`* * *` |busy student |keep track of all the tasks I have done | work on tasks that I have yet to do.
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |indecisive student |roll back and forth changes that I have done | track my ever-changing schedule.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |NUS student | view all the prerequisites for a specified module | plan my academic roadmap accordingly.
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |busy student |be reminded when I am nearing a deadline |be on top of all my assignments
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`*` |student |calculate my CAP easily |keep track of my progress in university.
 |=======================================================================
 
-_{More to be added}_
-
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
-
+(For all use cases below, the *System* is the `JARVIS` and the *Actor* is the `user`, unless specified otherwise)
 [discrete]
-=== Use case: Delete person
+=== Use case: Set tabs in finance tracker
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User inputs amount paid and the names of people who he paid for
+2. JARVIS calculates equal tab for all names including user
+3. JARVIS stores individual tabs for names input
+4. JARVIS prompts user that tabs have been added
+5. User requests to see list of debts owed to him
+6. JARVIS shows list of debts
 +
 Use case ends.
 
-*Extensions*
+[discrete]
+=== Use case: Marks task in planner as done
 
-[none]
-* 2a. The list is empty.
+*MSS*
+
+1. User requests to list tasks in planner
+2. JARVIS shows lists of tasks in planner
+3. User requests to mark a certain task as done
+4. JARVIS finds task and marks it as done
 +
 Use case ends.
 
+*Extensions*
+
 * 3a. The given index is invalid.
 +
 [none]
 ** 3a1. AddressBook shows an error message.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+[discrete]
+=== Use case: Undo previous command
+
+*MSS*
+
+1. User adds a project meeting into planner
```

Comment:
Seems too specific that the previous command has to be "adding a project meeting" when the use case is titled undo previous command